operator/v1/tests/clustercsidrivers.operator.openshift.io/VSphereDriverConfiguration.yaml

```diff
@@ -1,71 +0,0 @@
-apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
-name: "ClusterCSIDriver"
-crdName: clustercsidrivers.operator.openshift.io
-featureGate: VSphereDriverConfiguration
-tests:
-  onCreate:
-  - name: Should be able to create ClusterCSIDriver with snapshot options
-    initial: |
-      apiVersion: operator.openshift.io/v1
-      kind: ClusterCSIDriver
-      metadata:
-        name: csi.sharedresource.openshift.io
-      spec:
-        driverConfig:
-          driverType: vSphere
-          vSphere:
-            globalMaxSnapshotsPerBlockVolume: 1
-            granularMaxSnapshotsPerBlockVolumeInVSAN: 2
-            granularMaxSnapshotsPerBlockVolumeInVVOL: 3
-        logLevel: Normal
-        operatorLogLevel: Normal
-    expected: |
-      apiVersion: operator.openshift.io/v1
-      kind: ClusterCSIDriver
-      metadata:
-        name: csi.sharedresource.openshift.io
-      spec:
-        driverConfig:
-          driverType: vSphere
-          vSphere:
-            globalMaxSnapshotsPerBlockVolume: 1
-            granularMaxSnapshotsPerBlockVolumeInVSAN: 2
-            granularMaxSnapshotsPerBlockVolumeInVVOL: 3
-        logLevel: Normal
-        operatorLogLevel: Normal
-  - name: Should be able to create a minimal ClusterCSIDriver
-    initial: |
-      apiVersion: operator.openshift.io/v1
-      kind: ClusterCSIDriver
-      metadata:
-        name: csi.sharedresource.openshift.io
-      spec: {} # No spec is required for a ClusterCSIDriver
-    expected: |
-      apiVersion: operator.openshift.io/v1
-      kind: ClusterCSIDriver
-      metadata:
-        name: csi.sharedresource.openshift.io
-      spec:
-        logLevel: Normal
-        operatorLogLevel: Normal
-  - name: IBM Cloud CSIDriverType must have a defined IBM Cloud spec
-    initial: |
-      apiVersion: operator.openshift.io/v1
-      kind: ClusterCSIDriver
-      metadata:
-        name: csi.sharedresource.openshift.io
-      spec:
-        driverConfig:
-          driverType: IBMCloud
-    expectedError: "Invalid value: \"object\": ibmcloud must be set if driverType is 'IBMCloud', but remain unset otherwise"
-  - name: IBM Cloud spec must have an EncryptionKeyCRN defined
-    initial: |
-      apiVersion: operator.openshift.io/v1
-      kind: ClusterCSIDriver
-      metadata:
-        name: csi.sharedresource.openshift.io
-      spec:
-        driverConfig:
-          driverType: IBMCloud
-          ibmcloud: {}
-    expectedError: "spec.driverConfig.ibmcloud.encryptionKeyCRN: Required value, <nil>: Invalid value: \"null\": some validation rules were not checked because the object was invalid; correct the existing errors to complete validation"
```

Comment:
These tests are already present in the default set of tests?

Reply:
Yes, but the were duplicated in this file which is for the feature gate only I suppose - anyway, not applicable to latest version of this PR since it's not getting removed.